pdkconvertnoop.md

```diff
@@ -0,0 +1,33 @@
+# pdk convert noop
+
+As a user, I am simply intrigued by what it would take to convert my module to be managed by pdk. To this effect I have decided to go with a safe 'no-op' mode when it comes to running the pdk convert command.
```

Comment:
s/`managed`/`compatible`

Reply:
What term we should use is still in dispute, I'll bring it up at grooming.